actions/setup-sigstore-env/run-containers.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+docker_compose="docker compose"
+if ! ${docker_compose} version >/dev/null 2>&1; then
+    docker_compose="docker-compose"
+fi
+
+echo "setting up OIDC provider"
+pushd ./fakeoidc
+docker build . -t fakeoidc
+docker network ls | grep fulcio_default || docker network create fulcio_default --label "com.docker.compose.network=fulcio_default"
+docker network ps | grep fakeoidc || docker run -d --rm -p 8080:8080 --network fulcio_default --name fakeoidc fakeoidc
```

Comment:
I've already suggested replacing this but just for completeness `docker network ps` is not a thing and will always fail

Reply:
fixed